extension/docs/WebComponentsIntegrationGuide.md

```diff
@@ -128,13 +127,15 @@ The commercetools payment representation example:
       "key": "ctp-adyen-integration-web-components-payment-type"
     },
     "fields": {
```

Comment:
`Set getPaymentMethodsRequest custom field for a shopper in the Germany` should be moved to the before the `The commercetools payment representation example:` sentence.

Reply:
Sorry I don't understand this suggestion 😕

Reply:
`Set getPaymentMethodsRequest custom field for a shopper in the Germany` should be moved to line number 120

Reply:
https://github.com/commercetools/commercetools-adyen-integration/pull/542/commits/22027f2d8abce87d37e6bebda1643e1bd2113734